packages/augur-ui/src/modules/auth/containers/connect-account.ts

```diff
@@ -4,14 +4,18 @@ import {
   updateAuthStatus,
   IS_CONNECTION_TRAY_OPEN,
 } from 'modules/auth/actions/auth-status';
+import { updateMobileMenuState } from 'modules/app/actions/update-sidebar-status';
 
 const mapStateToProps = state => ({
   isLogged: state.authStatus.isLogged,
   userInfo: state.loginAccount.meta,
   isConnectionTrayOpen: state.authStatus.isConnectionTrayOpen,
+  isMobile: state.appStatus.isMobile,
```

Comment:
Lets get away from isMobile. I think we are moving toward Media queries. @phoebemirman  thoughts?

Reply:
agree

Reply:
removed